server/card.py

```diff
@@ -0,0 +1,3023 @@
+CARD_IMG_PNG = '''
```

Comment:
I love storing images like this. I think it's probably better as an asset file that gets loaded by python than as a python file, but that's really overkill for current stage. Just spitballing.